pytest-profiling/pytest_profiling.py

```diff
@@ -51,17 +53,85 @@ def pytest_sessionfinish(self, session, exitstatus):  # @UnusedVariable
             combined.dump_stats(self.combined)
             if self.svg:
                 self.svg_name = os.path.abspath(os.path.join(self.dir, "combined.svg"))
-                t = pipes.Template()
-                t.append("{} -f pstats $IN".format(self.gprof2dot), "f-")
-                t.append("dot -Tsvg -o $OUT", "-f")
-                t.copy(self.combined, self.svg_name)
+
+                # convert file <self.combined> into file <self.svg_name> using a pipe of gprof2dot | dot
+                # gprof2dot -f pstats prof/combined.prof | dot -Tsvg -o prof/combined.svg
+
+                # A/ Legacy: python pipes: does not seem to work on windows targets :(
```

Comment:
I'm happy for you to remove the commented-out version of the code and keep your pipes version - if it passes the build then we should be safe to keep just the one version. When merging the PR, don't squash it so that we can go back to this history if needs be.

Reply:
ok